src/Backend/Core/Engine/Authentication.php

```diff
@@ -280,6 +280,7 @@ public static function isAllowedModule($module)
         $modules = BackendModel::getModules();
         $alwaysAllowed = array('Core', 'Error', 'Authentication');
         $module = (string) $module;
+        $module = \SpoonFilter::toCamelCase($module);
```

Comment:
Can't you just cast the $module var to string in the toCamelCase call, instead of having two lines?